lib/relation-definition.js

```diff
@@ -1921,8 +1929,14 @@ EmbedsOne.prototype.update = function (targetModelData, cb) {
   if (embeddedInstance instanceof modelTo) {
     embeddedInstance.setAttributes(data);
     if (typeof cb === 'function') {
-      modelInstance.save(function(err, inst) {
-        cb(err, inst ? inst[propertyName] : embeddedInstance);
+      var context = { Model: modelTo, instance: embeddedInstance, hookState: {} };
+      modelTo.notifyObserversOf('before save', context, function(err) {
+        if (err) return cb(err);
+        modelInstance.save(function(err, inst) {
```

Comment:
Actually, I think that call is obsolete in `EmbedsOne.prototype.create` - a custom validation have been setup to handle this from the parent model. I prefer to remove it and let that validation handle it. I'll investigate.